app.rb

```diff
@@ -10,6 +10,15 @@
 class RelyingParty < Sinatra::Base
   use Rack::Session::Cookie, key: 'sinatra_sp', secret: SecureRandom.uuid
 
+  if LoginGov::Hostdata.in_datacenter?
+    configure do
+      enable :logging
+      file = File.new("/srv/sp-saml-sinatra/shared/log/production.log", 'a+')
```

Comment:
That directory isn't going to exist. The prefix is `/srv/sp-sinatra/`, since we never completed the second half of tasks necessary to rename the sample SPs.

Reply:
https://github.com/18F/identity-devops/blob/6fd7040412f75f2bcf7edfc328a6e81901471f6c/kitchen/cookbooks/login_dot_gov/recipes/sp_sinatra.rb#L5